docs/rules/no-invalid-this.md

```diff
@@ -11,12 +11,13 @@ Basically this rule checks whether or not a function which are containing `this`
 This rule judges from following conditions whether or not the function is a constructor:
 
 * The name of the function starts with uppercase.
+* The function is assigned to a variable the name starts with uppercase.
```

Comment:
Small grammar improvement if you're interested: "The function is assigned to a variable which starts with an uppercase letter."